glean_parser/schemas/pings.2-0-0.schema.yaml

```diff
@@ -96,6 +96,16 @@ additionalProperties:
             Interaction with `include_client_id`: `include_client_id` only takes
             effect when `metadata.include_info_sections` is `true`.
           type: boolean
+        ping_schedule:
+          title: Ping Schedule
+          description: |
+            An optional array of ping names. When one of the listed pings is
+            sent, then this ping will also be sent. A ping cannot list its own
+            name in `ping_schedule`.
+          type: array
+          items:
+            type: string
+            maxLength: 80
```

Comment:
The regex used to validate ping names found above restricts the length of ping names to 30 characters, so this `maxLength: 80` would never be realized.